File: Csharp/MiscExercises/TryBogus/Car.cs
namespace TryBogus;

public class Car
{
    public string Brand { get; set; } = string.Empty;

    public string Model { get; set; } = string.Empty;

    public double Price { get; set; }

    public override string ToString()
    {
        return $"{Brand} Car, {Model}, ${Price}";
    }
}

File: Tools/CMake/ModernCMakeForC++/chap4/project_structure/CarRental/src/lib_car/car.cpp
#include "include/car.hpp"
#include <iostream>

static const std::vector<Car> AVAILABLE_CARS{
    {"BMW", "525", 31.5},
    {"BYD", "Dolphin", 10.23},
    {"Volvo", "S90", 32.3}};

const std::vector<Car> &GetAvailableCars()
{
    return AVAILABLE_CARS;
}

void PrintCar(const Car &car)
{
    std::cout << "Car[" << car.Brand << "-" << car.Model << ", $" << car.Price << "]\n";
}
